dev/run-tests-jenkins.py

```diff
@@ -0,0 +1,256 @@
+        ERROR_CODES["BLOCK_PYTHON_STYLE"]: failure_result_note('Python style tests'),
+        ERROR_CODES["BLOCK_DOCUMENTATION"]: failure_result_note('to generate documentation'),
+        ERROR_CODES["BLOCK_BUILD"]: failure_result_note('to build'),
+        ERROR_CODES["BLOCK_MIMA"]: failure_result_note('MiMa tests'),
+        ERROR_CODES["BLOCK_SPARK_UNIT_TESTS"]: failure_result_note('Spark unit tests'),
+        ERROR_CODES["BLOCK_PYSPARK_UNIT_TESTS"]: failure_result_note('PySpark unit tests'),
+        ERROR_CODES["BLOCK_SPARKR_UNIT_TESTS"]: failure_result_note('SparkR unit tests'),
+        ERROR_CODES["BLOCK_TIMEOUT"]: failure_result_note('from timeout after a configured wait' +
+                                                          ' of \`' + tests_timeout + '\`')
+    }
+
+    if test_result == 0:
+        test_result_note = ' * This patch passes all tests.'
+    else:
+        test_result_note = failure_note_by_errcode[test_result]
+        send_archived_logs()
+
+    return [test_result_code, test_result_note]
+
+
+def main():
+    # Important Environment Variables
+    # ---
+    # $ghprbActualCommit
+    #   This is the hash of the most recent commit in the PR.
+    #   The merge-base of this and master is the commit from which the PR was branched.
+    # $sha1
+    #   If the patch merges cleanly, this is a reference to the merge commit hash
+    #     (e.g. "origin/pr/2606/merge").
+    #   If the patch does not merge cleanly, it is equal to $ghprbActualCommit.
+    #   The merge-base of this and master in the case of a clean merge is the most recent commit
+    #     against master.
+    ghprb_pull_id = os.environ["ghprbPullId"]
+    ghprb_actual_commit = os.environ["ghprbActualCommit"]
+    sha1 = os.environ["sha1"]
+
+    # Marks this build as a pull request build.
+    os.environ["AMP_JENKINS_PRB"] = "true"
+    build_display_name = os.environ["BUILD_DISPLAY_NAME"]
+    build_url = os.environ["BUILD_URL"]
+
+    commit_url = "https://github.com/apache/spark/commit/" + ghprb_actual_commit
+
+    # GitHub doesn't auto-link short hashes when submitted via the API, unfortunately. :(
+    short_commit_hash = ghprb_actual_commit[0:7]
+
+    # format: http://linux.die.net/man/1/timeout
+    # must be less than the timeout configured on Jenkins (currently 180m)
+    tests_timeout = "175m"
+
+    # Array to capture all test names to run on the pull request. These tests are represented
+    # by their file equivalents in the dev/tests/ directory.
+    #
+    # To write a PR test:
+    #   * the file must reside within the dev/tests directory
+    #   * be an executable bash script
+    #   * accept three arguments on the command line, the first being the Github PR long commit
+    #     hash, the second the Github SHA1 hash, and the final the current PR hash
+    #   * and, lastly, return string output to be included in the pr message output that will
+    #     be posted to Github
+    pr_tests = ["pr_merge_ability",
```

Comment:
Do you mind making the wrapping change here, so that this is written as 

```python
pr_tests = [
    "pr_merge_ability",
    "pr_public_classes",
    # DISABLED (pwendell) "pr_new_dependencies",
]
```